pom.xml

```diff
@@ -334,6 +335,10 @@
               <rules>
                 <enforceBytecodeVersion>
                   <maxJdkVersion>${compileSource}</maxJdkVersion>
+                  <ignoreClasses>
+                    <ignoreClass>module-info</ignoreClass>
+		    <ignoreClass>**.SubmissionPublisher*</ignoreClass>
```

Comment:
Can we scope this to specifically from Jersey? With a note that it has a multi-release jar?

Reply:
I renamed the pattern to **.glassfish.jersey.internal.jsr166.SubmissionPublisher*, I think it is enough to show that this is for jersey?